momentGW/pbc/rpa.py

```diff
@@ -0,0 +1,498 @@
+"""
+Construct RPA moments with periodic boundary conditions.
+"""
+
+import numpy as np
+from pyscf import lib
+
+from momentGW import mpi_helper, util
+from momentGW.pbc.tda import dTDA
+from momentGW.rpa import dRPA as MoldRPA
+
+# TODO: Check lack of Lai in the integrals
+
+
+class dRPA(MoldRPA, dTDA):
```

Comment:
do we want the MRO to be in this order? probably favour inheritence from `dTDA` over `MoldRPA` i would think